src/main/java/uk/gov/justice/digital/delius/controller/secure/StaffResource.java

```diff
@@ -18,36 +19,50 @@
-                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
-                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
-                    @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
-                    @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
-            })
-    @GetMapping(path = "/staff/staffCode/{staffCode}/managedOffenders")
-    public ResponseEntity<List<ManagedOffender>> getOffendersForResponsibleOfficer(
-            @ApiParam(name = "staffCode", value = "Delius officer code of the responsible officer", example = "SH0001", required = true)
-            @NotNull @PathVariable(value = "staffCode") final String staffCode,
-            @ApiParam(name = "current", value = "Current only", example = "false")  @RequestParam(name = "current", required = false, defaultValue = "false") final boolean current) {
-        return staffService.getManagedOffendersByStaffCode(staffCode, current)
-                .map(managedOffenders -> new ResponseEntity<>(managedOffenders ,OK))
-                .orElse(new ResponseEntity<>(NOT_FOUND));
-    }
+        private final StaffService staffService;
 
+        @ApiOperation(value = "Return list of of currently managed offenders for one responsible officer (RO)", notes = "Accepts a Delius staff officer code")
+        @ApiResponses(value = {
+                        @ApiResponse(code = 200, message = "OK", response = ManagedOffender.class, responseContainer = "List"),
+                        @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                        @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+                        @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
+                        @ApiResponse(code = 404, message = "Not found", response = ErrorResponse.class),
+                        @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class) })
+        @GetMapping(path = "/staff/staffCode/{staffCode}/managedOffenders")
+        public ResponseEntity<List<ManagedOffender>> getOffendersForResponsibleOfficer(
+                        @ApiParam(name = "staffCode", value = "Delius officer code of the responsible officer", example = "SH0001", required = true) @NotNull @PathVariable(value = "staffCode") final String staffCode,
+                        @ApiParam(name = "current", value = "Current only", example = "false") @RequestParam(name = "current", required = false, defaultValue = "false") final boolean current) {
+                return staffService.getManagedOffendersByStaffCode(staffCode, current)
```

Comment:
this looks like indenting at 8 spaces instead of 4?

Reply:
reformatted